cache.js

```diff
@@ -160,19 +160,31 @@ async function initAccountCache({chainName}) {
 }
 async function initTokenIdsCache() {
   let tokenIdOwners = await getTokenIDs(config.contractName)(config.network);
+  const tokenIDs = Object.keys(tokenIdOwners);
+  const uris = await getTokenURIs("rinkeby", tokenIDs);
+  uris.forEach(async uri => {
+    try {
+      await putRedisItem(`${uri.tokenId}`, `${uri.tokenURI}`, redisPrefixes[config.contractName] + 'uris');
```

Comment:
What is the point of this weird stringification? Are the types so unclear that there is a coersion needed?